lib/passport-wsfed-saml2/saml.js

```diff
@@ -119,7 +116,32 @@ SAML.prototype.validateSignature = function (xml, options, callback) {
   }
 };
 
+SAML.prototype.validateCertExpiration = function (samlAssertion, version) {
```

Comment:
This logic checks the certificate embedded in the assertion, but I believe we also have a situation where if there is no embedded certificate, we will check a configured certificate (`options.cert`)

https://github.com/auth0/passport-wsfed-saml2/blob/7803d873fa64f3ff292a61f68085b54e9ae778ad/lib/passport-wsfed-saml2/saml.js#L58-L62